framework/src/constraints/MortarSegmentInfo.C

```diff
@@ -119,12 +119,12 @@ MortarSegmentInfo::is_valid() const
 }
 
 bool
-MortarSegmentInfo::has_master() const
+MortarSegmentInfo::has_primary() const
```

Comment:
I guess this should be `hasPrimary()`

Reply:
remnants of mortar's libmesh origins :-)